index.html

```diff
@@ -51,8 +51,127 @@ <h1 class="intro-heading">Glad to see you!</h1>
                 </ul>
             </div>
         </div>
-        
     </section>
+
+    <section id="projects-section">
+        <article class="project-articles">
+            <div class="card">
+                <img src="/images/snapshot1.png">
+            </div>
+            <div class="textside">
+                <h2>Tonic</h2>
+                
+                    <ul class="card-ul">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized roads; no accounts or sign-up required</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+            </div>
+        </article>
+
+        <article class="project-articles">
+            <div class="textside">
+                <h2>Multi-Post Stories</h2>
+                
+                    <ul class="card-ul">
+                        <li>FACEBOOK</li>
+                        <li>Full Stack Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>Experiment content creation feature that allows users to add an existing story over the course of a day without spamming their friends</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on Rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
```

Comment:
- [ ] Kindly add the correct color, font-weight, and font size of all the texts in your project cards to match the figma design. Kindly look carefully to the CSS properties in the figma design 🥇 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219901286-4159728c-c7b6-486d-88e8-53681d0c83ed.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219901299-98b5d7a1-751f-4939-bdbb-37a26b310931.png)

Reply:
Done